calendar/src/main/java/org/exoplatform/platform/qa/ui/calendar/pageobject/EventManagement.java

```diff
@@ -711,7 +713,25 @@ public void saveQuickAddEvent() {
     ELEMENT_BUTTON_EVENT_SAVE.click();
     ELEMENT_BUTTON_EVENT_SAVE.waitUntil(Condition.disappears, Configuration.timeout);
   }
-
+  /**
+   * Select Previous and Next day
+   */
+  public void selectPreviousAndNextDayInTheEventToAdd(String previousDay, String nextDay){
+    info("Select Previous and Next day");
+    sleep(2000);
```

Comment:
sleep shouldn't be used, we should use waitUntil method instead.

Reply:
Fixed, done.